extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/ResteasyReactiveJacksonProcessor.java

```diff
@@ -205,6 +206,11 @@ private String getJacksonMessageBodyReader(boolean needsFullFeatureSet) {
                 : ServerJacksonMessageBodyReader.class.getName();
     }
 
+    @BuildStep
+    void reflection(BuildProducer<ReflectiveClassBuildItem> producer) {
+        producer.produce(ReflectiveClassBuildItem.builder(Cookie.class).methods().build());
```

Comment:
Is it really expected that you have to serialize a `Cookie`?

Reply:
There is no reason not to really. We don't to it ourselves, but I don't see any reason why we should make it more difficuklt

Reply:
OK, I consider it an internal data structure but yeah why not.

Reply:
Yeah, it's kinda grey :)